### DISCUSSION TITLE
blog/w/ask-new-bing

### DISCUSSION THREAD

**Discussion by giscus** [HK-SHAO/HK-SHAO.github.io#32] [General]
# blog/w/ask-new-bing

以下是我的一些提问，用作个人记录 图形学 Q 为了实现基于物理的渲染，路径追踪中大多数光线被俄罗斯轮盘赌结束掉了，或者最终没有击中光源，这些光线是没有贡献的，怎样让更多光线与材质表面交互后成为对图像有贡献的光线？; A 你好，这是必应。我可以帮你查找一些关于路径追踪的信息。😊 路径追踪是一种渲染技术，它通过模拟光线在场景中与物体表面的多次交互，来生成...

https://shao.fun/blog/w/ask-new-bing.html

**Comment by ghost** [comment]
烧风nb！！！
